src/geo/geohash.clj

```diff
@@ -194,22 +194,48 @@
 
 (defn- queue [] clojure.lang.PersistentQueue/EMPTY)
 
+(defn- geohashes-intersecting-matches!
+  "Given geohash and shape-relation, modify the matches list as necessary
+   :disjoint    no matches
+   :intersects  current geohash matches, but can't extend assertion to subdivisions
+   :within      same as within
```

Comment:
does this mean `same as :contains`?

Reply:
I actually meant `same as :intersects`, as in the treatment is the same. Sharp eye, I'll update.